### ISSUE TITLE
[CRITICAL] Remote Browser not sending correctly (and breaking bits) v1.4.9

### ISSUE THREAD

**Issue by J-eremy** [Sienci-Labs/gsender#540]
If you use the remote browser, load a file and send it, it sends bad gcode for starting the spindle and jogging and who knows what else. Below are screenshots of exactly what Im talking about. 

First I will give you the header of the gcode file.
```
G17
G21
G90
G53G0Z-5
G0X0.0000Y0.0000
S18000M3
G04 P3
```
This screenshot shows the console output of running the file with that header. I canceled the job after the spindle and dust collection turned on.
![image](https://github.com/user-attachments/assets/1f267112-43eb-45df-a152-1da1568f40de)

Next, I will connect to the computer remotely, then connect to the machine and do the same thing.
![image](https://github.com/user-attachments/assets/79797b9d-d309-46d8-8aa8-1dae550466be)
As you can see, it adds some information when remotely connecting where it is not if you're just using it on the machine that's connected directly to the CNC
`S18000M3 G4 Ptrue`

The only way to resolve this issue is to completely shut down the program, launch it, and NEVER connect to the remote interface at all for any reason.

This issue is related to the custom delay that was introduced in the latest version, or versions. Needless to say, it didn't start the spindle and then proceeded to snap a 45-dollar 1/8-inch Amana Spectra endmill. My post-processor already adds the delay and had no issues in previous versions either locally or remotely.
I'm using remotely rather liberally, as the remote device is in the same room, just has a much better monitor on it.

**This was a concern of mine when this feature was first announced, as crating and or modifying gcode should not be done by the sender at all, only the post processor.** _*sigh*_

Also, if you try and jog under the same circumstance from the remote end, you get this in the console.
![image](https://github.com/user-attachments/assets/ba74e68e-fff6-4098-b015-7770acf32499)
here I was just trying to raise the Z after the malfunction. This behavior only happens if you try and jog from the remote side, but device side it will jog, in this case, the Z.

TLDR: DON'T USE REMOTE MODE AT ALL.

**Comment by J-eremy** [comment]
@kglovern ref #505

**Comment by J-eremy** [comment]
> [@J-eremy](https://github.com/J-eremy) it's been a while since you reported it that I feel it's likely the new versions of gSender, such as 1.5.2, have addressed this issue; especially since the other issue you've linked was also fixed since then
> 
> Would you be able to give it a try and report back if you're still experiencing this?

No problem I'll check it out tonight. I haven't turned on remote mode at all after that. I'm currently running 1.5.2 so I'll definitely run some tests later tonight for you.
